readinessProbe.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -eo pipefail
+
+curl -H "Accept: application/json" http://127.0.0.1:31415/api/engine/status | jq -e '.started == true'
```

Comment:
I think you'll need some logic to determine the port. :-(

Might be worth re-factoring some of the stuff in `entrypoint.sh` into something that both scripts can source.